tests/unit/lib/queuePopulator/kafkaLogConsumer/LogConsumer.js

```diff
@@ -0,0 +1,219 @@
+const assert = require('assert');
+const sinon = require('sinon');
+const werelogs = require('werelogs');
+const { errors } = require('arsenal');
+const logger = new werelogs.Logger('KafkaLogConsumer');
+const ListRecordStream =
+    require('../../../../../lib/queuePopulator/KafkaLogConsumer/ListRecordStream');
+const LogConsumer =
+    require('../../../../../lib/queuePopulator/KafkaLogConsumer/LogConsumer');
+
+const kafkaConfig = {
+    hosts: 'localhost:9092',
+    topic: 'backbeat-oplog-topic',
+    groupId: 'backbeat-oplog-group',
+};
+
+const changeStreamDocument = {
+    ns: {
+        db: 'metadata',
+        coll: 'example-bucket',
+    },
+    documentKey: {
+        _id: 'example-key',
+    },
+    operationType: 'insert',
+    fullDocument: {
+        value: {
+            field: 'value'
+        }
+    }
+};
+const kafkaMessage = {
+    value: Buffer.from(JSON.stringify(changeStreamDocument)),
+    timestamp: Date.now(),
+    size: 2,
+    topic: 'oplog-topic',
+    offset: 1337,
+    partition: 0,
+    key: Buffer.from('key'),
+};
+
+describe('LogConsumer', () => {
+    let logConsumer;
+    beforeEach(() => {
+        logConsumer = new LogConsumer(kafkaConfig, logger);
+    });
+
+    describe('_getOffset', () => {
```

Comment:
This test is not really necessary, testing a getter gets the value you just set doesn't really help us much. As long as you use the getter in some other test it should be fine. I know it does stringify it but we don't need to test that javascript can stringify an object, also not entirely sure we should be stringifying a value like this. I would let our callee stringify it if they need it as a string.